_data/links.yml

```diff
@@ -23,6 +23,14 @@
   - 'background-size: 90%'
   - 'background-position-y: 25%'
   
+- name: EPITA Study Abroad
+  href: https://epitafr.sharepoint.com/sites/EPITAStudyAbroad
+  class: tile-medium text-dark
+  style:
+  - 'background-color: #fff'
+  - 'background-image: url(https://epitafr.sharepoint.com/sites/EPITAStudyAbroad/SiteAssets/__sitelogo__sans-titre.png)'
```

Comment:
This image is only accessible if you are logged into your EPITA account and should probably not be used as the logo for the tile. Maybe consider hosting it as part of the CDN solution that is used for some other tiles?

Reply:
Commit a8816e0 solves this.